design/tuple.md

```diff
@@ -90,7 +90,7 @@ Typecodes:
 &nbsp;`0x21` - double (64 bits)  
 &nbsp;`0x22` - long double (80 bits)  
 Length: 4 - 10 bytes  
-Test case: `pack( -42f ) == b'=\xd7\xff\xff'`  
+Test case: `pack( -42f ) == b'\x20\x3d\xd7\xff\xff'`
```

Comment:
The old value you had written here is what I get when I pack this value using Python. Is there a difference between some of our bindings here too?

Reply:
Oh wait, I understand now. The space character is missing (as reported in the source issue for this) and the '=' has been converted to hex. Looks good then.